src/stack-cache.test.js

```diff
@@ -1,8 +1,11 @@
 const core = require("@actions/core");
 const path = require("path");
 const process = require("process");
+const inputs = require("./inputs.js");
 const stackCache = require("./stack-cache.js");
 
+jest.mock("./inputs.js");
```

Comment:
:exploding_head: we could do a lot more in this test suite by mocking things (e.g. `@actions/core` and the like).

Reply:
Yeah, mocking an entire module is a pretty legit `jest` feature!